### DISCUSSION TITLE
Consider renaming `bud create` to `bud new` and `bud new` to `bud add`

### DISCUSSION THREAD

**Discussion by matthewmueller** [livebud/bud#139] [Proposals]
I find the differences between creating an application and scaffolding files within an app quite confusing. This would be a **breaking change**, so I think it makes sense to do before recording the next demo.

Currently it's:
- `bud create`: creates a new application
- `bud new`: scaffolds files within an application

I think it would be a bit less confusing if it was:
- `bud new`: creates a new application
- `bud add`: scaffolds files within an application

**Usage:**

```sh
bud new hackernews
bud add controller posts index show
```

**Other ideas:**

- Laravel uses `make` 
- Rails uses `g` / `generate`

**Questions:**
- Would scaffolding ever remove files? I can't think of any cases where this would be the case, but maybe certain advanced cases in other frameworks?

**Comment by sewera** [comment]
I think the migration will be less painful if it was something like:
- `bud create`, and
- `bud add`, with _deprecated_ alias `bud new`.

Then, you won't have to make a breaking change, should someone have already made some shell scripts for automation, even though those actions may seem non-automatable.

**Reply by matthewmueller** [reply]
Thanks for sharing @sewera! I'll definitely consider a 2-step approach. Might make sense to do that for v0.2 and then remove entirely for v0.3

Overall, I'm not too worried about making breaking changes during the v0.x releases. I definitely want to clean things up as I use Bud more.

**Comment by kokizzu** [comment]
init, add '__')

**Reply by matthewmueller** [reply]
Thanks for the feedback @kokizzu! For some reason I don't like `init` too much though I know a lot of other frameworks use it. I prefer using short, full words over abbreviations.
